.github/pull_request_template.md

```diff
@@ -16,6 +16,16 @@
   If it fixes an obvious bug, please tell users about the impact and effect of the problem.
 -->
 
+## What is the expected result?
+<!---
+  How one can checnk these changes after applying?  
+
+  Describe, what (resource, state, event, etc.) MUST or MUST NOT change/happen after applying these changes.
+-->
+
+## Checklist
+- [ ] The code is covered by unit tests (hooks, templates, matrix)
```

Comment:
```suggestion
- [ ] The code is covered by unit tests (hooks, templates, matrix)
```
This should be split into many checkboxes, and more checkboxes should be added.

Reply:
- Splitted into many checkboxes.
- Added more checkboxes (docs and manual test ina cluster)

@nabokihms

Reply:
I think there should be more, e.g., e2e tests are passed.

Reply:
Could you elaborate?  Add e2e and what else?

Reply:
As for me, that's all.

Reply:
Made changes.
@nabokihms